### PR TITLE
Test Case 12 : Running Reports

### PR DESCRIPTION
**Pull request by sherrif10**
cc @moshonk  @mozzy11

### CODE REVIEW

**Comment by mozzy11** [comment]
@sherrif10 , can you resolve these conflicts

**Comment by mozzy11** [comment]
@sherrif10 , can you resolve these conflicts

**Comment by sherrif10** [comment]
Sure thanks @mozzy11

File: src/features/isanteplus/reports.feature [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  Background:
+    Given User loads HomePage
+
+  @reports
+  Scenario: Running Reports
+  When    User clicks on Report App
+  And     User click the Report Name
+  #Then   User clicks on calendar icon
+  #And    User clicks on start date calendar
+  Then    User clicks on Run Button
```
Comment (mozzy11) [inline src/features/isanteplus/reports.feature:12]: @sherrif10    these steps seems incomplete as compared to the document

**Comment by sherrif10** [comment]
True @mozzy11  am just writing other features now thanks

**Comment by mozzy11** [comment]
Great Progress. you want this merged to continue ?? or i wait ??

**Comment by sherrif10** [comment]
> Great Progress. you want this merged to continue ?? or i wait ??

No please i want to finish it up quickly

**Comment by sherrif10** [comment]
Hello @mozzy11 , I need help according to the features am supposed to implement on this feature , am un able to query any report, do you mind helping me any query otherwise this is not getting implemented according to the steps an supposed to riight.

**Comment by sherrif10** [comment]
cc @mozzy11  , kindly review

**Comment by sherrif10** [comment]
cc @mozzy11

**Comment by sherrif10** [comment]
Hello @mozzy11  kindly review.Thanks

**Comment by sherrif10** [comment]
Do you consent with me we get this merged for now

**Comment by sherrif10** [comment]
Hello @mozzy11 is this worth merge

**Pull request merged**